calendar/Module.md

```diff
@@ -0,0 +1,94 @@
+## Overview
+Ballerina connector for Microsoft Outlook Calendar is connecting the Calendar API in Microsoft Graph via Ballerina language easily. 
+It provides capability to perform perform CRUD (Create, Read, Update, and Delete) operations on 
+[Event](https://docs.microsoft.com/en-us/graph/api/resources/event?view=graph-rest-1.0) &
+[Calendar](https://docs.microsoft.com/en-us/graph/api/resources/event?view=graph-rest-1.0) 
+stored in Microsoft OneDrive. 
+ 
+This module supports Microsoft Outlook Calendar API v1.0 version.
+ 
+### Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+* Create [Microsoft Outlook Account](https://outlook.live.com/owa/)
+* Obtaining tokens
+    Follow [this link](https://docs.microsoft.com/en-us/graph/auth-v2-user#authentication-and-authorization-steps) and obtain the client ID, client secret and refresh token.
+ 
+## Quickstart
+To use the outlook calendar connector in your Ballerina application, update the .bal file as follows:
+
+### Step 1: Import MS Outlook Calendar Package 
+import the ballerinax/microsoft.outlook.calendar module into the Ballerina project.
+```ballerina
+import ballerinax/microsoft.outlook.calendar;
+```
+### Step 2: Configure the connection to an existing Azure AD app
+You can now make the connection configuration using the OAuth2 refresh token grant config.
+```ballerina
+calendar:Configuration configuration = {
+    clientConfig: {
+        refreshUrl: <REFRESH_URL>,
+        refreshToken : <REFRESH_TOKEN>,
+        clientId : <CLIENT_ID>,
+        clientSecret : <CLIENT_SECRET>
+    }
+};
+calendar:Client calendarClient = check new(configuration);
+```
+### Step 3: Invoke  connector operation
+
+1. Now you can use the operations available within the connector. Note that they are in the form of remote operations.
+Following is an example on how to create a calendar using the connector.
+
+Create a calendar
```

Comment:
Let's either use the title or the description.

Reply:
Let's remove the title since the description explains the operation below.
/Thanks